### ISSUE TITLE
session management with Redis

### ISSUE THREAD

**Issue by salah2022-dvg** [CrowCpp/Crow#658]
Is it possible to handle sessions in crow cpp app using Redis

**Comment by gittiver** [comment]
Yes, but you have to do it on your own. For an example see https://github.com/CrowCpp/Crow/blob/master/examples/middlewares/example_session.cpp
